nagios_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+import (
+	"testing"
+)
+
+
+func TestGetNagiosTasksErrorWithMissingNagiosPod(t *testing.T) {
+	tasks := make(chan Task, 1)
+	GetNagiosTasks(tasks, []string{"project1"}, "basepath", func(project, resource, substr string) ([]string, error) {
+		return []string{}, nil
+	})
+	close(tasks)
```

Comment:
I'm not sure I follow you. The test and the GetNagiosTasks are both synchronous code on the same thread, how could the channel be written to after it is closed here?